getting-started/repl/user-snippets.md

```diff
@@ -0,0 +1,39 @@
+---
+title: User Snippets
```

Comment:
I agree that the "Predefined" prefix differentiates the snippets, but it doesn't provide much additional explanation of what is the kind of snippets. 
Since we are using the snippets that generate code only in the editor, maybe a better name could be Editor snippets or Code Editor snippets.

Reply:
Thank you both for the feedback! I initially used the name "User Snippets" as the menu in the REPL tool itself is called so. However, "Code Editor Snippets" name sounds more relevant in this scenario. I renamed the article and we may also consider renaming the menu in REPL.